### PR TITLE
Update em annotation product chart

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
https://tools.hmcts.net/jira/browse/EM-1936

Updated the repo to point to @hmctspublic and commented the validate job

### CODE REVIEW

File: azure-pipelines.yaml [hunk 1]
```diff
@@ -34,12 +35,12 @@ jobs:
...
+#    dependsOn: Validate
     pool:
       vmImage: 'Ubuntu 16.04'
     steps:
       - template: steps/charts/release.yaml@cnp-library
         parameters:
           chartName: em-annotation
           chartReleaseName: chart-em-annotation-release-1
-          chartNamespace: chart-tests
+          chartNamespace: evidence-mment
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml]: this is test namespace. better to let it be chart-tests

File: em-annotation/requirements.yaml [hunk 2]
```diff
@@ -1,19 +1,19 @@
 dependencies:
   - name: em-npa
     version: ~0.0.9
-    repository: '@hmcts'
+    repository: '@hmctspublic'
```
Comment (adusumillipraveen) [inline em-annotation/requirements.yaml:4]: Have you checked if all these charts are latest and available in hmctspublic?

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]

File: em-annotation/requirements.yaml (hunk 2 above)
Comment (yogesh-hullatti) [reply em-annotation/requirements.yaml:4]: Yes. Checked now. These are available on hmctspublic

File: azure-pipelines.yaml [hunk 3]
```diff
@@ -13,19 +13,20 @@ resources:
...
-      vmImage: 'Ubuntu 16.04'
-    steps:
-      - template: steps/charts/validate.yaml@cnp-library
-        parameters:
-          chartName: em-annotation
-          chartReleaseName: chart-em-annotation-ci-test
-          chartNamespace: chart-tests
-          helmInstallTimeout: "300"
+# IGNORE ANY VALIDATION FOR NOW - not sure how this will work... ci-values will be values - any point?
+#  - job: Validate
```
Comment (adusumillipraveen) [inline azure-pipelines.yaml]: You shouldn't remove this, this is supposed to test

**Review by markry-kainos — APPROVED** [review]

**Pull request merged**